static/app/views/eventsV2/table/quickContext.tsx

```diff
@@ -165,8 +190,24 @@ const ContextContainer = styled('div')`
   display: flex;
   flex-direction: column;
   margin: ${space(1.5)};
-  ${SidebarSection.Wrap} {
-    margin-bottom: 0;
+`;
+
+const IssueContextContainer = styled(ContextContainer)<{putMargin: boolean}>`
```

Comment:
I think there might be an opportunity for some clean up here so we don't need to add `putMargin` as a prop and add the `onCommitsLoaded` callback update some of these styled components!

```
const ContextContainer = styled('div')`
  display: flex;
  flex-direction: column;
  margin: ${space(1.5)};
  gap: ${space(1.5)};
`;

const IssueContextContainer = styled(ContextContainer)`
  ${SidebarSection.Wrap}, ${Panel} {
    margin: 0;
  }

  ${DataSection} {
    ${CauseHeader} {
      margin-top: 8px;
    }
    padding: 0;
  }

  ${StyledPanelItem} {
    justify-content: space-between;
  }
`;
```

Something like this might work to dynamically add `gap` between the rows. So instead of individually wrapping each of the bits in the `IssueContextContainer`, we can just return 

```
return (
    <IssueContextContainer>
      {renderStatus()}
      {renderAssigneeSelector()}
      {renderSuspectCommits()}
    </IssueContextContainer>
  );
```